.bazelrc

```diff
@@ -1,14 +1,13 @@
-# https://github.com/bazelbuild/bazel/issues/9451
-# ideally we want this, but it doesn't work...
-# build --cxxopt=-fno-canonical-system-headers
-# as a work around we use `sed` to "fix" the *.d files.
-
-build --crosstool_top=//toolchain:emscripten
-
-# Use --cpu as a differentiator.
-build --cpu=wasm32
-
-# Use the default Bazel C++ toolchain to build the tools used during the
-# build.
-
-build --host_crosstool_top=@bazel_tools//tools/cpp:toolchain
+build --cxxopt=-std=c++17
```

Comment:
I don't see `cpu=wasm32` flag? Should we add a config for it?

Reply:
Ah, so we still need x86 for unit tests.

Reply:
Yeah, the CPU option is added in the wasm binary transition rule: https://github.com/istio-ecosystem/wasm-extensions/pull/4/files#diff-1313d11afd1f5647109144e5f0283fa9R8